lib/Screens/SleepDiary/sleep_diary.dart

```diff
@@ -75,27 +76,27 @@ class _SleepPageState extends State<SleepPage> {
       value.docs.forEach((element) {
         if (element.id == data) {
           print("${element.id} é = a ${data}");
-          dataIgual = true;
+          //dataIgual = true;
         }
       });
       setState(() {
-        dataIgual = dataIgual;
+        //dataIgual = dataIgual;
         _loading = false;
       });
     });
   }
 
   Widget _buildQuest1() {
     return Container(
-      height: 50,
+      height: 80,
       margin: EdgeInsets.symmetric(vertical: 10.0),
       child: ListView(
         scrollDirection: Axis.horizontal,
         children: [
           Container(
             alignment: Alignment.center,
-            width: 200,
-            child: Text(
+            width: MediaQuery.of(context).size.width * .50,
```

Comment:
o que significa esse calculo?

Reply:
MediaQuery.of(context).size.width pega o tamanho da tela e faz vezes 50%. então esse container vai ter o tamanho de metade da tela do celular do usuário